civis/io/_files.py

```diff
@@ -196,6 +201,14 @@ def file_to_civis(buf, name=None, api_key=None, client=None, **kwargs):
     **kwargs : kwargs
         Extra keyword arguments will be passed to the file creation
         endpoint. See :func:`~civis.resources._resources.Files.post`.
+        In particular, ``expires_at`` can be specified for
+        the date and time the file will expire. If not specified, the file
+        will expire in 30 days. To keep a file indefinitely, specify ``None``.
+        To specify a date and time, format it by the ISO 8601 standard,
+        e.g., ``"2020-12-31"``, ``"2020-12-31T23:03:40Z"``,
+        and what the ``isoformat()`` method returns from
+        a :class:`datetime.date <datetime.date>`
+        or :class:`datetime.datetime <datetime.datetime>` object.
```

Comment:
I compiled the Sphinx docs locally and verified that the formatting and links would all be good:

![Screen Shot 2021-11-01 at 2 54 08 PM](https://user-images.githubusercontent.com/86482098/139858529-1335df1a-4d66-4df8-8ac8-3eb28df18576.png)